Source/pfile.cpp

```diff
@@ -102,7 +102,9 @@ BOOL pfile_create_player_description(char *dst, DWORD len)
 	_uiheroinfo uihero;
 
 	myplr = 0;
+#if !defined(__BIG_ENDIAN__)
 	pfile_read_player_from_save();

```

Comment:
https://github.com/diasurgical/devilutionX/blob/8e6276da584b332d81e79cf2cc01303e38ee5735/3rdParty/StormLib/src/SFileOpenFileEx.cpp#L299

This appears to be where things fail (having copied a valid save game from my PC)

Reply:
https://github.com/diasurgical/devilutionX/blob/8d907499296ba4612595272f0157886c2dd1dd62/3rdParty/StormLib/src/SBaseCommon.cpp#L703

x86:
```
szFileName: hero
dwHashIndexMask: 0x000007FF 2047
dwIndex: 0x000003D3 979
pHash->dwName1: 0x33AF486B 867125355
dwName1: 0x33AF486B 867125355
pHash->dwName2: 0xE3D1C651 -472791471
dwName2: 0xE3D1C651 -472791471
pHash->dwBlockIndex: 0x00000003 3
ha->dwFileTableSize: 0x00000800 2048
```

PowerPC:
```
szFileName: hero
dwHashIndexMask: 0x000007FF 2047
dwIndex: 0x000003D3 979
pHash->dwName1: 0xC7F14F34 -940486860
dwName1: 0x33AF486B 867125355
pHash->dwName2: 0x3278DBDF 846781407
dwName2: 0xE3D1C651 -472791471
pHash->dwBlockIndex: 0x9C181A00 -1676142080
ha->dwFileTableSize: 0x00000800 2048
```